pkg/agent/discovery/lvm.go

```diff
@@ -109,16 +111,22 @@ func (d *Discoverer) discoverVGs(newStatus *lssv1alpha1.NodeLocalStorageStatus,
 }
 
 func (d *Discoverer) createVG(vgname string, devices []string) error {
+	force := false
+	forceCreateVG := os.Getenv(localtype.EnvForceCreateVG)
```

Comment:
you use an environment value to input a parameter? 

I feel so weird. Why not use a command flag or something like that.